packages/core/src/ancillary/task-queue.ts

```diff
@@ -52,6 +56,8 @@ export class TaskQueue implements TaskQueueLike {
    * Size of the queue. Counts both deferred and currently running tasks.
    */
   get size(): number {
+    Metrics.observe(TASK_QUEUE_SIZE, this.#pq.size)
+    Metrics.observe(TASK_QUEUE_SIZE_PENDING, this.#pq.pending)
```

Comment:
These might be less useful because there will be many `TaskQueue` instances that we won't be able to differentiate since they're unnamed.

Looks like they're instantiated through `NamedTaskQueue`, so the metrics there should be sufficient.

Reply:
ok, removed!